support-frontend/assets/components/directDebit/directDebitProgressiveDisclosure/components/playback.jsx

```diff
@@ -53,20 +55,21 @@ function Playback(props: {
       </div>
 
       <div className="component-direct-debit-form__cta-container">
-        <button
-          className="component-direct-debit-form__cta component-direct-debit-form__cta--edit-button focus-target"
-          onClick={props.editDirectDebitClicked}
-        >
-          <span className="component-direct-debit-form__cta-text component-direct-debit-form__cta-text--inverse">Edit</span>
-        </button>
-        <button
-          id="qa-submit-button-2"
-          className="component-direct-debit-form__cta component-direct-debit-form__cta--confirm-button focus-target"
-          onClick={e => props.onSubmit(e)}
-        >
-          <span className="component-direct-debit-form__cta-text">{props.buttonText}</span>
-          <div className="component-direct-debit-form__arrow"><SvgArrowRightStraight /></div>
-        </button>
+        <ThemeProvider theme={buttonReaderRevenueAlt}>
+          <Button onClick={props.editDirectDebitClicked}>
+            <span className="component-direct-debit-form__cta-text component-direct-debit-form__cta-text--inverse">Edit</span>
+          </Button>
+        </ThemeProvider>
+        <ThemeProvider theme={buttonReaderRevenue}>
+          <Button
+            id="qa-submit-button-2"
+            className="component-direct-debit-form__cta component-direct-debit-form__cta--confirm-button focus-target"
+            onClick={e => props.onSubmit(e)}
+          >
+            <span className="component-direct-debit-form__cta-text">{props.buttonText}</span>
+            <div className="component-direct-debit-form__arrow"><SvgArrowRightStraight /></div>
+          </Button>
```

Comment:
Did the [button icon API](https://www.theguardian.design/2a1e5182b/p/435225-button/t/11af1c) not work out for you?

Reply:
Ah rubbish! No - I just missed it 😳 

I'll amend to use it.